### PR TITLE
feat: Add some details to the PR template

### PR DESCRIPTION
**Pull request by killthebuddh4**
# Issue link

N/A

# Description

* Added details to `pull_request_template.md`.

# How to test

* Nothing to test.

### CODE REVIEW

**Comment by killthebuddh4** [comment]
@andrewvdp Would like to get your thoughts on this PR template.

**Review by BrianS111 — APPROVED** [review]
Looks like a good template to go off of for now!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+
+- [] Chrome desktop
+- [] Firefox desktop
+- [] Safari desktop
+- [] Chrome Android
+- [] MetaMask browser Android
+- [] Safari iOS
+- [] MetaMask browser iOS
+
+**N.B.: Pull requests which don't include at a minimum the information specified
```
Comment (andrewvdp) [inline .github/pull_request_template.md:28]: Is the `minimum information specified` referring to one of the options from this list?
- The ideal option here is a list of existing, updated, or newly-created UI
  tests that cover the included functionality
- The next best option is probably a list of screenshots or videos
- The fallback option is a simple-but-detailed list of steps taken to verify changes

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+
+Changes were verified on the following platforms:
+
+- [] Chrome desktop
+- [] Firefox desktop
+- [] Safari desktop
+- [] Chrome Android
+- [] MetaMask browser Android
+- [] Safari iOS
+- [] MetaMask browser iOS
```
Comment (andrewvdp) [inline .github/pull_request_template.md:26]: This list could be very long. What do you think about changing to something like:
List browsers this change was tested for:
- Desktop <input>
- Android <input>
- Mobile <input>

File: .github/pull_request_template.md (hunk 1 above)
Comment (killthebuddh4) [reply .github/pull_request_template.md]: It's meant to refer to the whole PR template. Like "the minimum required by this template in general". I wasn't sure where to put that line. I have an idea though, will update.

File: .github/pull_request_template.md (hunk 2 above)
Comment (killthebuddh4) [reply .github/pull_request_template.md:29]: The point of the checklist isn't to ask developers which browsers were checked, it's for the developers to certify that the listed browsers were checked.

File: .github/pull_request_template.md (hunk 1 above)
Comment (killthebuddh4) [reply .github/pull_request_template.md]: @andrewvdp I moved the note to the top I slightly changed the wording.

**Review by andrewvdp — APPROVED** [review]

**Pull request merged**